src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,25 +55,34 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String recipient) {
+  public List<Message> getMessages(String name, String type) {
```

Comment:
nit: `String type` seems a little ambiguous here, I would rename it to something like `String queryType`

Reply:
changed to queryType